saleor/checkout/views/__init__.py

```diff
@@ -72,7 +72,7 @@ def login(request, checkout):
     Allows user to choose if he wants to login before checkout or continue
     as an anonymous user
     """
-    if request.user.is_authenticated:
+    if request.user.is_authenticated():
```

Comment:
This is backwards. Django seems to be moving away from this being a method (it's currently a property that returns a callable compatibility wrapper).